frontend/model/contracts/group.js

```diff
@@ -173,6 +186,7 @@ DefineContract({
     'gi.contracts/group': {
       validate: objectMaybeOf({
         invites: mapOf(string, inviteType),
+        joined_ms: number,
```

Comment:
remove (rely on existing `meta.createdDate`) (avoid creating new variables if they are not needed)

Reply:
I didn't remember we had `meta.createdDate`, will change!

Reply:
I didn't remember we had meta.createdDate, will change!